### PR TITLE
small bugfix for distancesensor handling

### PR DESCRIPTION
**Pull request by profff**
I'm working on a obstacle distance display and I found a small copy/paste bug in the distance sensors (#132) message handling 
im making a first PR just to correct this and will make a second PR for my display stuff if someone is intrested
thank's ( I'm a beginer in QT and in QGC organisation so if you find somthing wrong in my PR feel free to say it ( but gently ! ;) )

### CODE REVIEW

File: src/Vehicle/Vehicle.h [hunk 1]
```diff
@@ -76,9 +76,9 @@ class VehicleDistanceSensorFactGroup : public FactGroup
     Fact* rotationNone      () { return &_rotationNoneFact; }
     Fact* rotationYaw45     () { return &_rotationYaw45Fact; }
     Fact* rotationYaw90     () { return &_rotationYaw90Fact; }
-    Fact* rotationYaw135    () { return &_rotationYaw90Fact; }
+    Fact* rotationYaw135    () { return &_rotationYaw135Fact; }
     Fact* rotationYaw180    () { return &_rotationYaw180Fact; }
-    Fact* rotationYaw225    () { return &_rotationYaw180Fact; }
+    Fact* rotationYaw225    () { return &_rotationYaw225Fact; }
```
Comment (DonLakeFlyer) [inline src/Vehicle/Vehicle.h:81]: These are correct fixes.

File: src/Vehicle/Vehicle.h [hunk 2]
```diff
@@ -694,6 +694,7 @@ class Vehicle : public FactGroup
     Q_PROPERTY(FactGroup* setpoint          READ setpointFactGroup          CONSTANT)
     Q_PROPERTY(FactGroup* estimatorStatus   READ estimatorStatusFactGroup   CONSTANT)
     Q_PROPERTY(FactGroup* terrain           READ terrainFactGroup           CONSTANT)
+    Q_PROPERTY(FactGroup* distanceSensors   READ distanceSensorFactGroup    NOTIFY DistanceSensorReadingsChange) //####
```
Comment (DonLakeFlyer) [inline src/Vehicle/Vehicle.h:697]: This is incorrect. As is the additional signal. The distanceSensors point is constant.

**Comment by profff** [comment]
ok for wrong NOTIFY must I redoo the PR or can you just merge the .cc  "daf2477" commit  ?? sorry it's my first time contributing on qgc and in github

**Comment by DonLakeFlyer** [comment]
You can make the additional changes on the branch, squash them into a single commit and then push up the new changes.

**Pull request closed without merging**